Fix use of "$@" (see sh(1)).

There's no systemd on FreeBSD so start the session using $STARTUP which is
defined in 90-consolekit if ConsoleKit2 is installed.  By default this allows
local users to shutdown/reboot the machine and access devices like USB keys.

Also use dbus-run-session so libdbus doesn't have to autolauch the session bus
daemon on first use.  Autolaunched dbus daemons tend to linger and may only
exit if the X server exits.

--- data/scripts/Xsession.orig	2019-03-13 09:22:35 UTC
+++ data/scripts/Xsession
@@ -98,5 +98,5 @@ fi
 if [ -z "$*" ]; then
     exec xmessage -center -buttons OK:0 -default OK "Sorry, $DESKTOP_SESSION is no valid session."
 else
-    exec $@
+    exec $STARTUP /usr/bin/dbus-run-session --dbus-daemon=/usr/bin/dbus-daemon -- "$@"
 fi
